dali/kernels/signal/fft/fft_postprocess.cuh

```diff
@@ -69,6 +69,27 @@ struct norm2 {
   }
 };
 
+#if defined(__clang__) && defined(__CUDA__)
+
+template <typename T>
+__host__ const T& hostdev_max(const T& l, const  T& r) {
+  return std::max(l, r);
+}
+
+template <typename T>
+__device__ const T hostdev_max(const T& l, const  T& r) {
+  return ::max(l, r);
+}
+
+#else
+
+template <typename T>
+__host__ __device__ const T hostdev_max(const T& l, const T& r) {
+  return ::max(l, r);
+}
+
+#endif
```

Comment:
We have cuda_min/cuda_max. Currently they return reference, but that could be changed.